api/checks/update.go

```diff
@@ -201,7 +201,17 @@ func getDiffSummary(aeAPI shared.AppEngineAPI, diffAPI shared.DiffAPI, suite sha
 		PRNumbers:  suite.PRNumbers,
 	}
 
-	regressions := diff.Differences.Regressions()
+	var regressions mapset.Set
+	if aeAPI.IsFeatureEnabled("onlyChangesAsRegressions") {
```

Comment:
Hmm, would this mean that changing a support file and breaking lots of tests wouldn't count as a regression, or what's the definition of changes again? Is it that total number of tests have changed?

Can this change, and other things that feed into what the check result and text will be, be tested?

Reply:
Changing a file does not affect the WPT report output - it only changes which tests are considered affected (the input).

`C` / changed means "existed before, and after, and the result is different". So for some `a.html` test, adding a new subtest B would not count as "changed"; i.e. if a new subtest is failing, that wouldn't be counted as a regression.

Reply:
OK, so it's entirely based on the results, I should have been able to guess that.

Will a test changing its failure mode, going from FAIL to TIMEOUT for example, count as a regression?

Reply:
No; we just check for pass-or-not comparisons for the diff counts.